pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/worker/WorkerConfig.java

```diff
@@ -196,7 +196,7 @@
     private String functionWebServiceUrl;
     @FieldContext(
         category = CATEGORY_FUNC_METADATA_MNG,
-        doc = "The pulser binary service url that function metadata manager talks to"
+        doc = "The pulsar binary service url that function metadata manager talks to"
```

Comment:
```suggestion
        doc = "The Pulsar binary service URL that function metadata manager talks to"
```

Reply:
I updated all 4 of the `url` strings to `URL`. There are 7 `doc` fields in this `WorkerConfig` file that start with lower cased `pulsar` and there are no uses of upper cased `Pulsar`. Are all references to pulsar supposed to be capitalized?

Reply:
@michaeljmarshall If it refers to [Pulsar](https://pulsar.apache.org/docs/en/next/standalone/), we need to use the upper cased.

Reply:
Done.